Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -53,16 +60,40 @@ static CFStringRef __CFStringCreateUppercaseCopy(CFStringRef string, CFLocaleRef
 }
 
 /**
+ * Private helper that creates an array of CFString family names from a font collection
+ */
+CFArrayRef __DWriteCopyFontFamilyNamesFromCollection(IDWriteFontCollection* fontCollection) {
+    // Get the number of font families in the collection.
+    uint32_t count = fontCollection->GetFontFamilyCount();
+    woc::unique_cf<CFMutableArrayRef> fontFamilyNames(CFArrayCreateMutable(kCFAllocatorSystemDefault, count, &kCFTypeArrayCallBacks));
+    for (uint32_t i = 0; i < count; ++i) {
+        // Get the font family.
+        ComPtr<IDWriteFontFamily> fontFamily;
+        RETURN_NULL_IF_FAILED(fontCollection->GetFontFamily(i, &fontFamily));
```

Comment:
This is only being used by internal methods, which themselves don't return an HRESULT, so I don't think it's worth changing #WontFix